apps/studio/.storybook/main.ts

```diff
@@ -14,7 +14,7 @@ const config: StorybookConfig = {
   ],
 
   framework: {
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+    // @ts-expect-error type mismatch
```

Comment:
Why does this mismatch happens?

Reply:
types of storybook is stricter
`Type 'string' is not assignable to type 'string & FrameworkName'`